guide/payments/request.md

```diff
@@ -0,0 +1,471 @@
+
+## Requesting entry points
+
+When a user needs to request bitcoin, there are several ways in which this can be initiated including:
+
+- Generating single-use payment requests
+- Using reusable payment requests
+- Importing a withdraw request
+
+Below is an overview of these various entry points:
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/Home.png"
+   retina = "/assets/images/guide/payments/request/Home@2x.png"
+   layout = "shadow"
+   caption = "Home screens can provide quick access to entering request flows."
+   alt-text = "Screen showing home screen number pad entry."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/single-use-invoice.png"
+   retina = "/assets/images/guide/payments/request/single-use-invoice@2x.png"
+   layout = "shadow"
+   caption = "Single-use payment requests, like invoices, are useful for in-person payments."
+   alt-text = "Screen showing single-use invoice."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/my-contact-card.png"
+   retina = "/assets/images/guide/payments/request/my-contact-card@2x.png"
+   layout = "shadow"
+   caption = "Reusable payment requests, like Lightning addresses, are useful for receiving recurring payments."
+   alt-text = "Screen showing reusable Lightning address."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/withdraw-request.png"
+   retina = "/assets/images/guide/payments/request/withdraw-request@2x.png"
+   layout = "shadow"
+   caption = "A withdraw request is another way a user can request bitcoin."
+   alt-text = "Screen showing withdraw request UI."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+## Single-use payment requests
+
+The most common way bitcoin is requested today is in the form of single-use payment requests. For Lightning, a standard [BOLT11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) invoice is used. For on-chain, [addresses of various types]({{ "/guide/glossary/address/" | relative_url }}) are used. Addresses can be reused though this is not advised due to privacy concerns. single-use invoices and addresses need to be easily and conveniently generated within a wallet application.
+
+### Amount
+
+When generating a single-use payment request, users need to be able to add an amount to be paid.  Entering an amount from the home screen can speed up the requesting flow. Alternatively, users can add an amount to an already generated payment request. Entering an amount needs to be clear to prevent users entering incorrect amounts.
```

Comment:
Who is the user here? Receiver/Sender? It's may not be clear entirely who's who to someone reading this.

Reply:
Good point, will update!